docs/v2-roadmap.rst

```diff
@@ -1,83 +1,226 @@
-v2.0 Roadmap
-============
+v2.0 Migration Guide
+====================
 
-.. important::
+**urllib3 v2.0 is now available!** Read below for how to get started and what is contained in the new major release.
 
-   We're seeking `sponsors and supporters for urllib3 v2.0 on Open Collective <https://github.com/sponsors/urllib3>`_.
-   There's a lot of work to be done for our small team and we want to make sure
-   development can get completed on-time while also fairly compensating contributors
-   for the additional effort required for a large release like ``v2.0``.
+**🚀 Migrating from 1.x to 2.0**
+--------------------------------
+
+We're maintaining **functional API compatibility for most users** to make the
+migration an easy choice for almost everyone. Most changes are either to default
+configurations, supported Python versions, or internal implementation details.
+So unless you're in a specific situation you should notice no changes! 🎉
+
+.. note::
+
+  If you have difficulty migrating to v2.0 or following this guide
+  you can `open an issue on GitHub <https://github.com/urllib3/urllib3/issues>`_
+  or reach out in `our community Discord channel <https://discord.gg/urllib3>`_.
+
+
+Timeline for deprecations and breaking changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The 2.x initial release schedule will look like this:
+
+* urllib3 ``v2.0.0-alpha1`` will be released in November 2022. This release
+  contains **minor breaking changes and deprecation warnings for other breaking changes**.
+  There may be other pre-releases to address fixes before v2.0.0 is released.
+* urllib3 ``v2.0.0`` will be released in early 2023 after some initial integration testing
```

Comment:
I was hoping late 2022 could be achievable? I won't be really available after December 23rd though! So yeah early 2023 is the more reasonable choice.